pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/MapValueTransformFunction.java

```diff
@@ -29,20 +31,25 @@
 
 
 /**
- * map_value(keyColName, 'keyName', valColName)
+ * The MAP_VALUE transform function takes 3 arguments: keyColumn, keyValue, valueColumn, where keyColumn and valueColumn
+ * are dictionary-encoded multi-value columns, and keyValue must be a literal (number or string). In order to make
+ * MAP_VALUE transform function work, the keyValue provided must exist in the keyColumn. To ensure that, the query can
+ * have a filter on the keyColumn, e.g. SELECT mapValue(key, 'myKey', value) FROM myTable WHERE key = 'myKey'.
  */
 public class MapValueTransformFunction extends BaseTransformFunction {
+  public static final String FUNCTION_NAME = "mapValue";
```

Comment:
why camel case. its common to use _ as the separator in sql udf's. See presto udf's for example https://prestodb.github.io/docs/current/functions/string.html

Reply:
We already have `valueIn`, `timeConvert`, `dateTimeConvert`. Prefer keeping them the same way.